xcode/Subconscious/Shared/Components/Feed/FeedEmptyView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  FeedEmptyView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 13/10/2023.
+//
+
+import SwiftUI
+
+struct FeedEmptyView: View {
+    var onRefresh: () -> Void
+
+    var body: some View {
+        GeometryReader { geom in
+            ScrollView {
+                VStack {
+                    Spacer()
+                    HStack {
+                        Spacer()
+                        VStack(spacing: AppTheme.unit * 6) {
+                            Image(systemName: "sparkles")
+                                .font(.system(size: 64))
+                            VStack(spacing: AppTheme.unit) {
+                                Text("Your feed is empty")
+                                Text("When you follow others their notes will appear here")
```

Comment:
We should use punctuation here because we're providing two lines.

Your feed is empty.
When you follow others, their notes will appear here.